lib/model/folder.go

```diff
@@ -317,7 +316,7 @@ func (f *folder) getHealthErrorAndLoadIgnores() error {
 	}
 	if f.Type != config.FolderTypeReceiveEncrypted {
 		if err := f.ignores.Load(".stignore"); err != nil && !fs.IsNotExist(err) {
-			return errors.Wrap(err, "loading ignores")
+			return fmt.Errorf("loading ignores: %w", err)
```

Comment:
Should this and others be changed from `loading` to simply `load` to be consistent?

Reply:
While we're at it, yeah I'd prefer that.

Reply:
Yeah, no, this is about the way errors are generated. I'd welcome a review of returned errors (and have a number of changes queued that add additional context strings), but this PR is not that.